cmd/drone-docker/main.go

```diff
@@ -192,6 +192,17 @@ func main() {
 			Usage:  "docker email",
 			EnvVar: "PLUGIN_EMAIL,DOCKER_EMAIL",
 		},
+		cli.StringFlag{
+			Name:   "commit.branch",
+			Value:  "master",
+			Usage:  "git commit branch",
+			EnvVar: "DRONE_COMMIT_BRANCH",
+		},
+		cli.StringFlag{
+			Name:   "default.branch",
```

Comment:
let's call this `repo.branch` which is consistent with some of the other plugins and the [plugin starter](https://github.com/drone/drone-plugin-starter/blob/master/main.go)

Reply:
Done. https://github.com/drone-plugins/drone-docker/pull/160/commits/0927e34a03293409b1de12926974fd125083e7ce